data_processing/tests/test_eia_functions.py

```diff
@@ -0,0 +1,614 @@
+import pytest
+from eia import fuel_type_code
+from eia import mover_database
+from eia import generation
+from eia import university_top_producers
+from eia import energy_type_breakdown
+from eia import university_top_renewables
+from eia import usage
+from eia import capacity_factors
+from eia import university_capacity
+from eia import sources
+from eia import split_dictionary
+from eia import plot_data
+from eia import plot_energy_change
+
+path_arg = './'
```

Comment:
```suggestion
path_arg = './data_processing/tests'
```

This fixes the tests, finally. The relative paths are a little confusing. The path to the sample data must be *relative to ``eia.py``* because that's where the function is actually called. Just "commit suggestion" and we'll be good to go!